admin_site/templates/system/image_versions.html

```diff
@@ -0,0 +1,83 @@
+{% extends "site_with_navigation.html" %}
+
+{% block specific_title %}
+Versioner
+{% endblock %}
+
+{% block javascripts %}
+
+{% endblock %}
+
+{% block image_versions_active %}
+class="active"
+{% endblock %}
+
+{% block specific_content %}
+
+<ul class="sublevelnav">
+  {% if site_allowed %}
+    {% for maj_v in major_versions %}
+    <li {% if request.resolver_match.url_name == '/site/{{ site.url }}/image-versions/{{ maj_v }}' %}class="active"{% endif %}>
+      <a class="list-item" href="/site/{{ site.url }}/image-versions/{{ maj_v }}"><i class="icon-tag"></i> v{{ maj_v }}</a>
+    </li>
+    {% endfor %}
+  {% endif %}
+</ul>
+ 
+<div class="container-fluid main sublevelmain">
+
+  {% if site_allowed %}
+    {% with minor_versions.all|dictsortreversed:"img_vers"|first as first_obj %}
+      <h2 class="divideheader"><em>Nyeste underversion af v{{ first_obj.img_vers|slice:":1" }}</em></h2>
+      <ul class="unstyled">
+        <li><b>VERSION:</b> {{ first_obj.img_vers }}</li>
+        <li><b>RELEASE DATO:</b> {{ first_obj.rel_date }}</li>
+        <li><b>OPERATIVSYSTEM:</b> {{ first_obj.os }}</li>
+        <li><b>UDGIVELSESNOTER:</b> {{ first_obj.rel_notes }}</li>
+        <li>
+          <b>DOWNLOAD: </b>
+          <button class="btn">
+            <i class="fa fa-home">
+              <a href="downsload_link/{{ first_obj.img_vers }}">v{{ first_obj.img_vers }}.iso</a>
```

Comment:
Shouldn't this be `<a href="{{ first_obj.img_path }}">`, as in the loop below?

Reply:
Yes.Thanks for pointing that out.